src/metabase/api/dashboard.clj

```diff
@@ -1010,7 +1010,22 @@
                                                                (map (fn [card]
                                                                       (if-let [real-tab-id (get old->new-tab-id (:dashboard_tab_id card))]
                                                                         (assoc card :dashboard_tab_id real-tab-id)
-                                                                        card))))
+                                                                        card)))
+                                                               true
+                                                               (map (fn [dashcard]
+                                                                      (cond
+                                                                        (pos-int? (:action_id dashcard))
```

Comment:
Isn't this gonna *boom* on non action dashcards? I think you need to handle non-int and 0 first.

Reply:
No, I'm being silly, and this is fine.